ubuntu/mambaforge_deprecation.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [[ "$GITHUB_ACTIONS" == "true" ]]; then
+    echo "::warning title=Mambaforge is now deprecated!::Future Miniforge releases will NOT build Mambaforge installers. We advise you switch to Miniforge at your earliest convenience. More details at https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/."
+else
+    echo "!!!!!! Mambaforge is now deprecated !!!!!"
+    echo "Future Miniforge releases will NOT build Mambaforge installers."
+    echo "We advise you switch to Miniforge at your earliest convenience."
+    echo "More details at https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/."
+fi
+
+case $(date +%F) in 
+    # Brownouts
+    2024-10-01|2024-10-15|2024-11-01|2024-11-10|2024-11-20|2024-11-30|2024-12-05|2024-12-10|2024-12-15|2024-12-20|2024-12-25|2024-12-30|2024-12-31|2025-*)
+        exit 1
+    ;;
+    *)
+        echo "Sleeping for 300s..."
+        for i in {1..10}; do print "."; done
```

Comment:
```suggestion
        for i in {1..300}; do sleep 1; printf "."; done
        echo
```

Did you mean that @hmaarrfk? I applied the suggestion before but I was only seeing the first edit with `sleep 300`, not this `for` loop.

Reply:
yes sorry for the delay